lib/pages/chat/events/html_message.dart

```diff
@@ -29,10 +30,24 @@ class HtmlMessage extends StatelessWidget {
     this.linkStyle,
     this.emoteSize,
     this.bottomWidgetSpan,
+    this.topWidgetIntegration,
   }) : super(key: key);
 
   @override
   Widget build(BuildContext context) {
+    if (topWidgetIntegration != null) {
```

Comment:
Why don't you build an Stateless Widget instead of change code of old one like this ? like `PreviewLinkHtmlMessage`

Reply:
this widget will end untouched